test/parallel/test-path-join.js

```diff
@@ -7,56 +7,56 @@ const failures = [];
 const backslashRE = /\\/g;
 
 const joinTests = [
-  [ [path.posix.join, path.win32.join],
-    // Arguments                     result
-    [[['.', 'x/b', '..', '/b/c.js'], 'x/b/c.js'],
-     [[], '.'],
-     [['/.', 'x/b', '..', '/b/c.js'], '/x/b/c.js'],
-     [['/foo', '../../../bar'], '/bar'],
-     [['foo', '../../../bar'], '../../bar'],
-     [['foo/', '../../../bar'], '../../bar'],
-     [['foo/x', '../../../bar'], '../bar'],
-     [['foo/x', './bar'], 'foo/x/bar'],
-     [['foo/x/', './bar'], 'foo/x/bar'],
-     [['foo/x/', '.', 'bar'], 'foo/x/bar'],
-     [['./'], './'],
-     [['.', './'], './'],
-     [['.', '.', '.'], '.'],
-     [['.', './', '.'], '.'],
-     [['.', '/./', '.'], '.'],
-     [['.', '/////./', '.'], '.'],
-     [['.'], '.'],
-     [['', '.'], '.'],
-     [['', 'foo'], 'foo'],
-     [['foo', '/bar'], 'foo/bar'],
-     [['', '/foo'], '/foo'],
-     [['', '', '/foo'], '/foo'],
-     [['', '', 'foo'], 'foo'],
-     [['foo', ''], 'foo'],
-     [['foo/', ''], 'foo/'],
-     [['foo', '', '/bar'], 'foo/bar'],
-     [['./', '..', '/foo'], '../foo'],
-     [['./', '..', '..', '/foo'], '../../foo'],
-     [['.', '..', '..', '/foo'], '../../foo'],
-     [['', '..', '..', '/foo'], '../../foo'],
-     [['/'], '/'],
-     [['/', '.'], '/'],
-     [['/', '..'], '/'],
-     [['/', '..', '..'], '/'],
-     [[''], '.'],
-     [['', ''], '.'],
-     [[' /foo'], ' /foo'],
-     [[' ', 'foo'], ' /foo'],
-     [[' ', '.'], ' '],
-     [[' ', '/'], ' /'],
-     [[' ', ''], ' '],
-     [['/', 'foo'], '/foo'],
-     [['/', '/foo'], '/foo'],
-     [['/', '//foo'], '/foo'],
-     [['/', '', '/foo'], '/foo'],
-     [['', '/', 'foo'], '/foo'],
-     [['', '/', '/foo'], '/foo'],
-    ],
+  [[path.posix.join, path.win32.join],
```

Comment:
This change seems irrelevant to the pull request. Can you remove unnecessary and out of context changes from this PR?